prov/rxd/src/rxd_av.c

```diff
@@ -320,9 +320,12 @@ static int rxd_av_lookup(struct fid_av *av, fi_addr_t fi_addr, void *addr,
 	fi_addr_t dg_fiaddr;
 
 	rxd_av = container_of(av, struct rxd_av, util_av.av_fid);
+	if (fi_addr >= ofi_av_size(&rxd_av->util_av))
```

Comment:
I don't think this does anything. It doesn't hurt but I don't think it's needed and still has an edge case at index 0 when nothing has been inserted